spec/index.bs

```diff
@@ -1304,8 +1307,8 @@ no validation errors are raised, most promises resolve normally, etc.
                 Note: {{GPUDeviceLostInfo/message}} should not disclose unnecessary user/system
                 information and should never be parsed by applications.
         </div>
-    1. Complete any outstanding {{GPUBuffer/mapAsync()}} steps.
-    1. Complete any outstanding {{GPUQueue/onSubmittedWorkDone()}} steps.
+
+    1. Complete any outstanding steps that are waiting until device <dfn dfn>becomes lost</dfn>.
```

Comment:
That is the one in mapAsync, which I did update.

For most of the others that I was able to find, I don't think they should point to [=becomes lost=]. I'm using that to refer to the "event" of becoming lost for the purposes of hooking "listeners".

The most ambiguous are:
"when the device becomes invalid (is lost or destroyed)" (1 time)
"The device may become lost if shader execution does not end" (3 times)
But I think it's correct for those to point to [=lose the device=], because of:
"Any time the user agent needs to revoke access to a device, it calls [=lose the device=]"

Maybe [=becomes lost=] needs a better name? Not sure what though.

(I'll note we already have two different links for these things, [=lose the device=] and [$invalid|lost$], however I think the way we use them right now is reasonable and consistent.)

Reply:
SGTM. I don't know what we could change "becomes lost" to that would be significantly clearer, so it's fine by me to keep it as-is.